tests/test_core_schedule.py

```diff
@@ -851,6 +881,22 @@ def test_removing_service_updates_vehicles(schedule):
     assert schedule.vehicles == {}
 
 
+def test_multiple_services_are_no_longer_present_in_schedule_after_removing(schedule_graph):
+    s = Schedule(_graph=schedule_graph)
+    assert set(s.service_ids()) == {'service1', 'service2'}
+    assert set(s.route_ids()) == {'2', '1', '4', '3'}
+    s.remove_services(['service1', 'service2'])
+    assert set(s.service_ids()) == set()
+    assert set(s.route_ids()) == set()
+
+
+def test_removing_multiple_services_updates_changelog(schedule_graph):
+    s = Schedule(_graph=schedule_graph)
+    s.remove_services(['service1', 'service2'])
+    list(s.change_log().iloc[-2:][['change_event', 'old_id']].to_records()) == [(0, 'remove', 'service1'), (1, 'remove', 'service2')]
```

Comment:
Send out the assertion search squad! 😄 

I had a mess about with this test, or at least the other one like it, and when I turned this statement into an assertion it failed. I think it was to do with the types of the objects in the two lists we're comparing; one was holding tuples and the other was holding numpy record objects, so they _looked_ the same, but failed equality comparison. I'm guessing there is a quick and easy "pandonic" way to make sure our expectation list has records, not tuples (or the actual list has tuples rather than records)?